docs/06-concepts/02-models.md

```diff
@@ -226,9 +226,31 @@ Supports any string value, e.g., `"This is a string"` or `'This is a string'`.
 Example:
 
 ```yaml
-stringDefault: String, default='This is a string', defaultModel="This is a string", defaultDatabase="This is a string"
+stringDefault: String, default='This is a string', defaultModel="This is a string", defaultPersist="This is a string"
 ```
 
+### UuidValue
+
+Supports two types of default values:
+
+1. **UUID String Values**:
+   - A valid UUID string can be specified, e.g., `'550e8400-e29b-41d4-a716-446655440000'`.
```

Comment:
Question: Only v4 UUIDS are expected right? Maybe we should have a note on that.